Core/Tools/Compress/Compress.cpp

```diff
@@ -20,9 +20,10 @@
 #include <cstdio>
 #include "Lib/BaseType.h"
 #include "Compression.h"
-#include "../CRCDiff/debug.h"
+// TheSuperHackers @compile feliwir 11/04/2025 Use the release DEBUG_LOG in debug aswell
+//#include "../CRCDiff/debug.h"
```

Comment:
Why is this changed? Can we avoid this for a clean move?

Perhaps this CRCDiff dependency needs to be cleaned up first in a prior change?

Reply:
Can we make that a separate change to clean up the dependency? Why is it commented out?

Reply:
Decoupling moved to #658